uswds/uswds
# v2: Update additional dependencies

## Description

Updates and locks USWDS v2 dependencies. Uses `npm shrinkwrap` command to lock all dependencies.

### About shrinkwrap

```zsh
npm shrinkwrap
```

> This command repurposes package-lock.json into a publishable npm-shrinkwrap.json or simply creates a new one. The file created and updated by this command will then take precedence over any other existing or future package-lock.json files. For a detailed explanation of the design and purpose of package locks in npm, see [package-locks](https://docs.npmjs.com/cli/v6/configuring-npm/package-locks).

**Source:** https://docs.npmjs.com/cli/v6/commands/npm-shrinkwrap

## Additional information

Updated patch and minor dependencies before running shrinkwrap. Below is a table documenting the history of these updates.

| Dependency                   | Old     | [790bc36] | [868fa6d] | [ce3b97b]|
| :--------------------------- | :------ | :-------- | --------- | -------- |
| autoprefixer                 | 10.4.8  | -         | 10.4.11   | 10.4.14  |
| axe-core                     | 4.4.3   | -         | -   | 4.6.3  |
| @babel/preset-env            | 7.18.10 | -         | 7.19.1    | 7.20.2   |
| @frctl/fractal               | 1.5.13  | 1.5.14    | -         | 1.5.15   |
| @frctl/mandelbrot            | 1.10.1  | 1.10.2    | -         | 1.10.3  |
| @frctl/nunjucks              | 2.0.14  | 2.0.15    | -         | -        |
| @types/node                  | 16.11.6 | -         | 18.7.18   | 18.15.3  |
| chrome-remote-interface      | 0.31.3  | -         | -         | 0.32.1   |
| classlist-polyfill           | 1.0.3   | -         | -         | 1.2.0    |
| eslint                       | 8.21.0  | 8.23.0    | 8.23.1    | 8.36.0   |
| eslint-config-prettier       | 8.5.0   |           |           | 8.7.0    |
| eslint-plugin-import         | 2.26.0  | -         | -         | 2.27.5   |
| eslint-plugin-no-unsanitized | 4.0.1   | -         | -         | 4.0.2    |
| gulp-replace                 | 1.1.3   | -         | -         | 1.1.4    |
| postcss                      | 8.4.14  | 8.4.16    | -         | 8.4.21   |
| postcss-csso                 | 6.0.0   | 6.0.1     | -         | -        |
| prettier                     | 2.7.1   | -         | -         | 2.8.4    |
| sass                         | 1.54.1  | 1.54.8    | 1.54.9    | 1.59.3   |
| snyk                         | 1.982.0 | 1.996.0   | 1.1006.0  | 1.1119.0 |
| stylelint                    | 14.9.1  | 14.11.0   | -         | -        |
| typescript                   | 4.7.4   | 4.8.2     | 4.8.3     | -        |
| yargs                        | 17.5.1  | -         | -         | 17.7.1   |

## How to test

1. Verify `package-lock.json` has been replaced by `shrinkrwap.json`
2. Run `npm install`
3. Dependencies should be locked to versions specified in `npm-shrinkwrap.json`

---

Before you hit Submit, make sure you’ve done whichever of these applies to you:
